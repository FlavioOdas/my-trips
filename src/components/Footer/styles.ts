import styled from 'styled-components'

export const Footer = styled.footer`
  display: flex;
  justify-content: center;
  align-items: center;
  align-self: end;
  width: 100%;

  margin-top: 3rem;

  font-size: 2rem;
`

export const FooterLink = styled.a`
  color: #fff;
`
