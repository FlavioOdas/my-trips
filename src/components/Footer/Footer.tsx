import React from 'react'

import * as S from './styles'

const Footer = () => {
  return (
    <S.Footer>
      <S.FooterLink
        href="https://github.com/FlavioOdas"
        title="My github"
        target="_blank"
      >
        Github
      </S.FooterLink>
    </S.Footer>
  )
}

export default Footer
