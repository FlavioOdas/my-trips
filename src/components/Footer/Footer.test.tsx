import { render } from '@testing-library/react'
import React from 'react'

import Footer from './Footer'

describe('Footer', () => {
  it('should render correclty', () => {
    const component = render(<Footer />)

    expect(component.asFragment).toMatchSnapshot()
  })
})
