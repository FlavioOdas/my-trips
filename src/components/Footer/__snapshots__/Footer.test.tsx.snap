// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correclty 1`] = `[Function]`
